lib/cellect/server/adapters/panoptes.rb

```diff
@@ -1,76 +1,71 @@
-require 'pg'
-require 'connection_pool'
+require 'active_record'
 
 module Cellect
   module Server
     module Adapters
+      module PanoptesAssociation
+        class Workflow < ActiveRecord::Base
+          has_many :subject_sets
+        end
+
+        class SubjectSet < ActiveRecord::Base
+          belongs_to :workflow
+          has_many :set_member_subjects
+        end
+
+        class SetMemberSubject < ActiveRecord::Base
+          belongs_to :subject_set
+        end
+
+        class UserSeenSubject < ActiveRecord::Base
+        end
+      end
+
       class Panoptes < Default
         def initialize
-          @pg ||= ConnectionPool.new(size: ENV.fetch('PG_POOL_SIZE', 25).to_i) do
-            PG.connect connection_options
-          end
+          ActiveRecord::Base.establish_connection(**connection_options)
```

Comment:
I'm pretty sure AR creates a pool with 5 connections by default. We can configure more connections with the `pool:` key.